html/changelog.html

```diff
@@ -173,6 +173,10 @@ <h3 class="author">RaveRadbury updated:</h3>
 				<li class="bugfix">Binary Encryption keys work in pAIs that have downloaded the encryption key software.</li>
 			</ul>
 			<h3 class="author">TemporalOroboros updated:</h3>
+			<ul class="changes bgimages16">
+				<li class="rscadd">The supermatter can now consume miasma in the air to power itself.</li>
+			</ul>
+			<h3 class="author">Thalpy updated:</h3>
```

Comment:
I assume this is an error, there shouldn't be a manual addition to the changelog.

Reply:
Pesky master